interim_results.py

```diff
@@ -0,0 +1,128 @@
+import pyaudio
+import asyncio
+import sys
+import websockets
+import time
+import json
+import beepy
+import shutil
+import argparse
+
+SILENCE_INTERVAL = 2.0
+
+FORMAT = pyaudio.paInt16
+CHANNELS = 1
+RATE = 16000
+CHUNK = 8000
+
+terminal_size = shutil.get_terminal_size()
+
+audio_queue = asyncio.Queue()
+
+def callback(input_data, frame_count, time_info, status_flag):
+    audio_queue.put_nowait(input_data)
+    return (input_data, pyaudio.paContinue)
+
+async def run(key):
+    clock_cursor = 0.
+    audio_cursor = 0.
+    transcript_cursor = 0.
```

Comment:
is measuring latency necessary for this demo?

Reply:
Ah, no. I can pull it out.